build_tools/build_ext.py

```diff
@@ -81,13 +82,17 @@ def _build_cmake(self, build_dir: Path, install_dir: Path) -> None:
             build_command.append(str(max_jobs))
 
         # Run CMake commands
+        start_time = time.time()
```

Comment:
Consider using time.perf_counter() for benchmarking instead of time.time(). The time.perf_counter() function is monotonic, providing more accurate timing measurements. You can use [time.get_clock_info](https://docs.python.org/3/library/time.html#time.get_clock_info) to check the differences between these functions.

Reply:
Thanks. I will have a look.